README.md

```diff
@@ -154,6 +154,31 @@ Now your local CKAN deployment will use the `localhost:54392/ckan.2.10.4` image.
 
 When creating fine-grained API tokens, ensure that read/write permissions for PRs and Content are allowed (this will allow for PR and commit creation) and that CI user is used to create the PRs.
 
+## Ephemeral cluster
+
+An ephemeral cluster is now available to test changes to the CKAN and datagovuk charts and infrastructure without impacting the other stable EKS environments.
+
+To access the ephemeral cluster from your terminal you will need to log in to the govuk-test account
+
+```sh
+eval $(gds-cli aws govuk-test-admin -e --art 8h)
+```
+
+and follow these [steps](https://docs.publishing.service.gov.uk/kubernetes/get-started/access-eks-cluster/#access-a-cluster-for-the-first-time) to switch to the test environment.
+
+If there is an existing ephemeral datagovuk cluster running you can delete the datagovuk cluster by deleting `dgu-app-of-apps` in the ephemeral argo website and recreate the cluster by running this command on the terminal under the `charts` directory (the environment variable passed in should match your ephemeral cluster name):
+
+```sh 
+helm upgrade -n cluster-services datagovuk-argo-bootstrap argo-bootstrap --set environment=eph-aaa113
+```
+
+To test your changes on the ephemeral cluster without merging them into the `main` branch:
+
+- update the `targetRevision` in the `ckan-application.yaml` and/or `datagovuk-application.yaml` and push this change up to your branch onto Github
+- if you have deleted the `dgu-app-of-apps` module then run the `helm upgrade` command as above
```

Comment:
Is this right? Deleting it in ArgoCD or in your repository?

Reply:
I've added it again as there is another step with the `helm upgrade` so its slightly different